listenbrainz/webserver/views/api.py

```diff
@@ -665,6 +665,64 @@ def get_service_details(user_name):
     return jsonify({'user_name': user_name, 'services': services})
 
 
+@api_bp.route("/tags/", methods=['GET', 'OPTIONS'])
+@crossdomain
+@ratelimit()
+def get_tags_dataset():
+    """ Get the recordings with the specified tag. """
```

Comment:
We need to finalize the url for this endpoint before merging. Also add some more docs here methinks.

Reply:
Agreed.